packages/design/src/Form/components/Fieldset/index.tsx

```diff
@@ -12,8 +13,7 @@ const Fieldset: PatternComponent<FieldsetProps> = props => {
           {props.legend}
         </legend>
       )}
-
-      {props.children}
+      {renderPromptComponents(props.context, props.childComponents)}
```

Comment:
All the components that have children have been updated to call a function to render the child prompt component parts. This will enable rendering components with alternate preview props in the edit UI, which will be necessary for the logic UI.